### ISSUE TITLE
onItemSelect is not working in ListTemplate for android

### ISSUE THREAD

**Issue by shubhadeep-foodhub** [birkir/react-native-carplay#156]
**Describe the bug**
const template = new ListTemplate({
            sections: [
                {
                    header: 'HEADER',
                    items: [
                        {
                            text: 'Item 1',
                        },
                    ],
                },
            ],
            title: 'Test Android Auto',
            async onItemSelect(e) {
                console.log("test clicked", e);
            },
        });
        CarPlay.setRootTemplate(template);

When testing the list ListTemplate, the onItemSelect function is not working. 

**To Reproduce**
1. click on list item

**Expected behavior**
Should fire a onItemSelect event

**Android Auto (please complete the following information):**
 - Device: [e.g. Oneplus 6 and  DHU]
 - Android Version: [11]
 - RNCarPlay version [2.4.0-beta.2 - Android Auto]

**Comment by birkir** [comment]
Awesome, first Android bug (I expect loads...)
I will look at this

**Comment by shubhadeep-foodhub** [comment]
Yes, it has lots of bugs which is causing frequent app crashes. waiting for a stable release.

**Comment by tanquetav** [comment]
List view items only trigger events if they are browsable items:

```
const template = new ListTemplate({
            sections: [
                {
                    header: 'HEADER',
                    items: [
                        {
                            text: 'Item 1',
                            browsable: true,
                        },
                    ],
                },
            ],
            title: 'Test Android Auto',
            async onItemSelect(e) {
                console.log("test clicked", e);
            },
        });
        CarPlay.setRootTemplate(template);
```

**Comment by Deepak-Foodhub** [comment]
> List view items only trigger events if they are browsable items:
> 
> ```
> const template = new ListTemplate({
>             sections: [
>                 {
>                     header: 'HEADER',
>                     items: [
>                         {
>                             text: 'Item 1',
>                             browsable: true,
>                         },
>                     ],
>                 },
>             ],
>             title: 'Test Android Auto',
>             async onItemSelect(e) {
>                 console.log("test clicked", e);
>             },
>         });
>         CarPlay.setRootTemplate(template);
> ```

It works. Thanks @tanquetav
